srcpkgs/textosaurus/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'textosaurus'
+
+pkgname=textosaurus
+version=0.9.7
+revision=1
+build_style=qmake
+configure_args=build.pro
+hostmakedepends="git pkg-config qt5-svg-devel qt5-qmake"
+makedepends="qt5-tools qt5-svg-devel"
+depends="qt5-svg"
+short_desc="Simple cross-platform text editor based on Qt and Scintilla"
+maintainer="Leandro Vital <leavitals@gmail.com>"
+license="GPL-3"
```

Comment:
Please use SPDX-compliant license identifiers. This should GPL-3.0-only (I believe)